app/Models/BuildFailure.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Builder;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\Relations\HasOne;
+
+/**
+ *
+ * @property int $buildid
+ * @property int $detailsid
+ * @property string $sourcefile
+ * @property int $newstatus
+ *
+ * @mixin Builder<BuildFailure>
+ */
+class BuildFailure extends Model
+{
+    protected $table = 'buildfailure';
+
+    public $timestamps = false;
+
+    protected $fillable = [
+        'buildid',
+        'detailsid',
+        'sourcefile',
+        'newstatus',
+    ];
+
+    protected $casts = [
+        'buildid' => 'integer',
+        'detailsid' => 'integer',
+        'newstatus' => 'integer',
+    ];
+
+    /**
+     * @return BelongsTo<Build, self>
+     */
+    public function build(): BelongsTo
+    {
+        return $this->belongsTo(Build::class, 'buildid');
+    }
+
+    /**
+     * @return HasOne<BuildFailureDetails>
+     */
+    public function configure(): HasOne
+    {
+        return $this->hasOne(BuildFailureDetails::class, 'id', 'detailsid');
+    }
```

Comment:
This doesn't seem right.  Shouldn't this be `details()` or similar?

Reply:
thanks for catching this